src/views/Correction.vue

```diff
@@ -39,46 +39,70 @@
     />
 
     <StaffPaymentDialog
+      v-model:staffPaymentData="staffPaymentData"
       attach="#correction"
       :dialog="staffPaymentDialog"
-      :staffPaymentData.sync="staffPaymentData"
       :loading="filingPaying"
       @exit="staffPaymentDialog=false"
       @submit="onClickFilePay(true)"
     />
 
     <!-- Initial Page Load Transition -->
     <v-fade-transition>
-      <div class="loading-container" v-show="showLoadingContainer">
+      <div
+        v-show="showLoadingContainer"
+        class="loading-container"
+      >
         <div class="loading__content">
-          <v-progress-circular color="primary" :size="50" indeterminate />
-          <div class="loading-msg">{{loadingMessage}}</div>
+          <v-progress-circular
+            color="primary"
+            :size="50"
+            indeterminate
+          />
+          <div class="loading-msg">
+            {{ loadingMessage }}
+          </div>
         </div>
       </div>
     </v-fade-transition>
 
     <!-- Main Body -->
-    <v-container id="correction-container" class="view-container" v-show="dataLoaded">
+    <v-container
+      v-show="dataLoaded"
+      id="correction-container"
+      class="view-container"
+    >
       <v-row>
-        <v-col cols="12" lg="9">
+        <v-col
+          cols="12"
+          lg="9"
+        >
           <article id="correction-article">
             <!-- Page Title -->
             <header>
-              <h1 id="correction-header">Correction &mdash; {{title}}</h1>
-              <p class="text-black">Original Filing Date: {{originalFilingDate}}</p>
+              <h1 id="correction-header">
+                Correction &mdash; {{ title }}
+              </h1>
+              <p class="text-black">
+                Original Filing Date: {{ originalFilingDate }}
+              </p>
             </header>
 
             <!-- Detail -->
             <section>
               <header>
-                <h2 id="correction-step-1-header">1. Detail</h2>
+                <h2 id="correction-step-1-header">
+                  1. Detail
+                </h2>
                 <p>Enter a detail that will appear on the ledger for this entity.</p>
-                <p class="black--text mb-0">{{defaultComment}}</p>
+                <p class="black--text mb-0">
```

Comment:
Here's yet another way to set a color (see https://v2.vuetifyjs.com/en/styles/colors/).

In fact, this is probably wrong and should be a `grayX`, but ignore that for now.